packages/bpk-component-horizontal-nav/stories.scss

```diff
@@ -38,4 +38,10 @@ $custom-color: $bpk-color-sagano;
     height: 10rem;
     border-right: 0.1rem lightgray solid;
   }
+
+  &__light-appearance-wrapper {
```

Comment:
This is fine for the story, but when you do the docs, please use the `dark` example property instead of using a custom style
```
  {
    id: 'light',
    title: 'Light',
    blurb: [
      <Paragraph>The light badge is a gray color as shown below.</Paragraph>,
    ],
    dark: true,
    examples: [
      <BpkBadge type={BADGE_TYPES.light} className={badgeClassName}>
        Apples
      </BpkBadge>,
      ...
```

Reply:
Will do